functional-samples/tutorial.reading-time/images/README.md

```diff
@@ -0,0 +1,15 @@
+# Tutorial: Run scripts on every page
```

Comment:
We should remove this duplicate in the images directory

Reply:
Removed!